docs/src/pages/_meta.json

```diff
@@ -3,6 +3,7 @@
   "getting-started": "Getting Started",
   "nextjs": "Next.js Setup",
   "solid": "SolidStart Setup",
+  "backend-adapters": "Backend Adapters",
```

Comment:
next and solidstart can go under backend adapters too, and then they can link to `/api-reference/react` (and solid)

Reply:
we can do this restructuring later though

Reply:
Don't worry about structure too much until we have the community repo setup, I'll do an overhaul